### ISSUE TITLE
Uncaught (in promise) TypeError when loading a new page of images in console.

### ISSUE THREAD

**Issue by EllangoK** [AlUlkesh/stable-diffusion-webui-images-browser#28]
When loading a page of images I always get a `Uncaught (in promise) TypeError` in the javascript console.

In chrome it is
`index.4395ab38.js:76 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'forEach') at st`

In firefox it is
`Uncaught (in promise) TypeError: xt.data is undefined`

The stack trace is not too useful either
![image](https://user-images.githubusercontent.com/23466035/217980177-58bb7e11-8412-49f9-ab9c-aa55f5cfed51.png)

This error has no effect as far as I can tell as the images still load and everything.

**To Note:** This has been existent before the db update and I think before the fork in general. It is not recent.

**Comment by AlUlkesh** [comment]
Seems to come from here:
```
            function st(xt) {
                xt.data.forEach((dt, Gt) => {
                    if (typeof dt == "object" && dt !== null && dt.__type__ === "update") {
                        for (const [zt, Lt] of Object.entries(dt))
                            zt !== "__type__" && n(11, L[jt[Gt]].props[zt] = Lt, L);
                        n(9, m)
                    } else
                        n(11, L[jt[Gt]].props.value = dt, L)
                })
            }
```
which is deep in gradio territory.

I think I found the issue on their side:
https://github.com/gradio-app/gradio/issues/2907

So, while a workaround might be possible by coding outputs, even if we don't need them, I'd rather wait for gradio to fix it on their side in this case.

**Comment by EllangoK** [comment]
Nice that you know where its from, I was trying to figure it out myself early but it all seemed like builtins or whatever. And yeah I agree I don't think we need to do anything about as like I said everything still works. 
I think it is good to note it down though anyways in case anyone else might submit this error.

**Comment by AlUlkesh** [comment]
Fixed with 19dff48
